### PR TITLE
OpenShift support

### PR DESCRIPTION
**Pull request by matheusfm**
## Description
These changes improve the OpenShift support

## How has this been tested?
- Installing in a OpenShift cluster

## Checklist
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/undistro/.github/labels?q=Type%3A)
- [x] I have documented my code (if applicable)
- [ ] My changes are covered by tests

### CODE REVIEW

File: docs/configuration/private-registries/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Private Registries
```
Comment (knrc) [inline docs/configuration/private-registries/index.md:1]: Change this to Authenticated Registries, they don't need to be private

File: mkdocs.yml [hunk 2]
```diff
@@ -86,11 +86,13 @@ nav:
           - Compute resources: configuration/resources.md
           - Suspending scans: configuration/suspend-scan.md
           - Retain issues: configuration/retain-issues.md
+          - HTTPS proxy: configuration/https-proxy.md
           - Ignore unfixed vulnerabilities: plugins/trivy/#large-vulnerability-reports
-          - HTTPS Proxy: configuration/https-proxy.md
-          - Scanning Images hosted in AWS ECR: configuration/aws-elastic-container-registry.md
-          - Scanning Images hosted in Azure ACR: configuration/acr.md
           - Vulnerability scan timeout: plugins/trivy/#scan-timeout
+          - Private registries:
```
Comment (knrc) [inline mkdocs.yml:92]: We should differentiate between private registries (for AWS/Azure) and Authenticated Registries.

File: charts/zora/templates/plugins/trivy.yaml [hunk 3]
```diff
@@ -58,6 +60,10 @@ spec:
         --all-namespaces \
         --scanners=vuln \
         -f=json \
+        --cache-dir=/tmp/trivy-cache \
```
Comment (knrc) [inline charts/zora/templates/plugins/trivy.yaml:63]: Are you adding a PVC/PV in this PR?  Are you keeping them separate?

File: docs/configuration/private-registries/index.md (hunk 1 above)
Comment (matheusfm) [reply docs/configuration/private-registries/index.md]: Done

File: charts/zora/templates/plugins/trivy.yaml (hunk 3 above)
Comment (matheusfm) [reply charts/zora/templates/plugins/trivy.yaml:63]: Done

File: mkdocs.yml (hunk 2 above)
Comment (matheusfm) [reply mkdocs.yml:93]: Done

File: charts/zora/templates/plugins/trivy.yaml (hunk 3 above)
Comment (matheusfm) [reply charts/zora/templates/plugins/trivy.yaml:63]: I'll do it in a separate PR

**Pull request merged**